packages/playground/src/pages/pipeline-edit-page.tsx

```diff
@@ -275,7 +275,7 @@ export default function PipelineEditPage() {
   )
 
   return (
-    <div style={{ height: '650px' }}>
+    <div className="h-[calc(100vh-144px)]">
```

Comment:
can we remove this wrapper div and add the correct height on `Container.Root` directly by passing `className` to it?

Reply:
Solved in the new commit